src/OmniSharp.Abstractions/project.json

```diff
@@ -33,14 +31,17 @@
         "System.Xml.Linq": ""
       }
     },
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.3": {
+      "imports": [
+        "dotnet5.4",
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
       "dependencies": {
-        "NETStandard.Library": "1.0.0-rc2-23910",
-        "System.Dynamic.Runtime": "4.0.11-rc2-23911",
-        "System.Diagnostics.Process": "4.1.0-rc2-23911",
-        "System.IO.FileSystem.Watcher": "4.0.0-rc2-23911",
-        "System.Threading.Thread": "4.0.0-rc2-23911"
+        "NETStandard.Library": "1.5.0-*",
```

Comment:
I was planning on using the new `SetPackageVersions` command to pin it to a certain version defined as good.